tests/flow/test_relation_patterns.py

```diff
@@ -141,8 +141,149 @@ def test04_variable_length_labeled_nodes(self):
         actual_result = redis_graph.query(query)
         self.env.assertEquals(actual_result.result_set, expected_result)
 
+    # Test traversals that don't specify an edge direction.
+    def test05_bidirectional_traversals(self):
+        query = """MATCH (a)-[:e]-(b) RETURN a.val, b.val ORDER BY a.val, b.val"""
+        actual_result = redis_graph.query(query)
+        # Each relation should appear twice with the source and destination swapped in the second result.
+        expected_result = [['v1', 'v2'],
+                           ['v2', 'v1'],
+                           ['v2', 'v3'],
+                           ['v3', 'v2']]
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        # Test undirected traversals with a referenced edge.
+        query = """MATCH (a)-[e:e]-(b) RETURN ID(e), a.val, b.val ORDER BY a.val, b.val"""
+        actual_result = redis_graph.query(query)
+        expected_result = [[0, 'v1', 'v2'],
+                           [0, 'v2', 'v1'],
+                           [1, 'v2', 'v3'],
+                           [1, 'v3', 'v2']]
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        # Test 0-hop undirected traversals.
+        query = """MATCH (a)-[*0]-(b) RETURN a.val, b.val ORDER BY a.val, b.val"""
```

Comment:
Consider moving all variable length traversals to tests/flow/test_variable_length_traversals.py
or create a new flow test file dedicated for bi-directional patterns